net/acme/files/run.sh

```diff
@@ -214,12 +254,18 @@ issue_cert()
         return 1
     fi
 
-    if [ "$update_uhttpd" -eq "1" ]; then
+    if [ -e /etc/init.d/uhttpd ] && [ "$update_uhttpd" -eq "1" ]; then
         uci set uhttpd.main.key="$STATE_DIR/${main_domain}/${main_domain}.key"
         uci set uhttpd.main.cert="$STATE_DIR/${main_domain}/fullchain.cer"
         # commit and reload is in post_checks
     fi
 
+    if [ -e /etc/init.d/nginx ] && [ "$update_nginx" -eq "1" ]; then
+        sed -i "s/ssl_certificate\ .*/ssl_certificate $STATE_DIR/${main_domain}/fullchain.cer;" /etc/nginx/nginx.conf
+        sed -i "s/ssl_certificate_key\ .*/ssl_certificate_key $STATE_DIR/${main_domain}/${main_domain}.key;" /etc/nginx/nginx.conf
```

Comment:
The sed expressions are invalid; you need to escape the slashes in the substitution